### DISCUSSION TITLE
TLS 1.3 support

### DISCUSSION THREAD

**Discussion by jcarlson-ars** [rabbitmq/rabbitmq-dotnet-client#1049] [Q&A]
RabbitMQ says it supports TLS 1.3, but requires .Net 4.7, the latest version of this client is 4.6.1.  Is there a plan to upgrade to support TLS 1.3?

**Comment by stebet** [comment]
You can target .NET Framework 4.7+ and the client will then be able to use TLS 1.3. The target framework of the client auto-upgrades to the one targeted by your app.

**Reply by jcarlson-ars** [reply]
Thanks, so guess I will need to update to .Net core 5.0, using 3.1 now.  Everything I initially saw said core 3.1 would support when windows supported, so I made an assumption, not smart.  Just did some digging and see they released it with 5.0.  So I'm out of luck then I guess till I can update to 5.0

**Reply by stebet** [reply]
No. .NET Core supports TLS 1.3 just fine. .NET Core is not the same runtime as .NET Framework, it is much newer.

**Reply by stebet** [reply]
See here for an explanation https://auth0.com/blog/navigating-dotnet-maze/
